Day8/puzzle.py

```diff
@@ -0,0 +1,46 @@
+from copy import deepcopy
```

Comment:
D E E P C O P Y

I actually got the right answer just doing a shallow copy, but I tested and saw that the original was being changed too. Do not want!